protected/humhub/modules/admin/assets/AdminPendingRegistrationsAsset.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2017 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\admin\assets;
+
+use yii\web\AssetBundle;
```

Comment:
Use the `humhub\components\assets\AssetBundle` instead

Reply:
Done!